util/find-doc-nits

```diff
@@ -203,6 +203,57 @@ sub check_head_style {
     }
 }
 
+# This is from http://man7.org/linux/man-pages/man7/man-pages.7.html
+my %preferred_words = (
+        'bitmask'	=> 'bit mask',
+        'builtin'	=> 'built-in',
+        'epoch'	        => 'Epoch',
+        'file name'	=> 'filename',
+        'file system'	=> 'filesystem',
+        'host name'	=> 'hostname',
+        'i-node'	=> 'inode',
+        'lower case'	=> 'lowercase',
+        'lower-case'	=> 'lowercase',
+        'non-zero'	=> 'nonzero',
+        'path name'	=> 'pathname',
+        'pseudo-terminal' => 'pseudoterminal',
+        'reserved port'	=> 'privileged port',
+        'system port'	=> 'privileged port',
+        'realtime'	=> 'real-time',
+        'real time'	=> 'real-time',
+        'runtime'	=> 'run time',
+        'saved group ID'=> 'saved set-group-ID',
+        'saved set-GID'	=> 'saved set-group-ID',
+        'saved user ID'	=> 'saved set-user-ID',
+        'saved set-UID'	=> 'saved set-user-ID',
+        'set-GID'	=> 'set-group-ID',
+        'setgid'	=> 'set-group-ID',
+        'set-UID'	=> 'set-user-ID',
+        'setuid'	=> 'set-user-ID',
+        'super user'	=> 'superuser',
+        'super-user'	=> 'superuser',
+        'super block'	=> 'superblock',
+        'super-block'	=> 'superblock',
+        'time stamp'	=> 'timestamp',
+        'time zone'	=> 'timezone',
+        'upper case'	=> 'uppercase',
+        'upper-case'	=> 'uppercase',
+        'useable'	=> 'usable',
+        'userspace'	=> 'user space',
+        'user name'	=> 'username',
+        'zeroes'	=> 'zeros'
```

Comment:
Hehehe, is this a US English vs The Queen's English thing?

Reply:
I know policy is UK English, so are there anything to fix in that list?  man7.org says to use US English.

Reply:
I'm not about sure the _zeroes_ to _zeros_ change.  The [Oxford dictionary](https://www.lexico.com/en/definition/zero) notes that _zeros_ is plural and _zeroes_ is a verb.

> Make sure your function zeroes memory by filling it with zeros. :grin:

Reply:
Ah, considering we might potentially use the word in both forms, this line should go.

Reply:
I'm ambivalent, frankly.  Please do fix the errors, but consider giving a warning instead of an error on `zeroes`

Reply:
Another option is to use "zeroise" (sic) instead of "zeroes".  But I'll fix the errors and mull it over a bit.

Reply:
I think that _zeroise_ is evil.

Reply:
I used "clear" instead.  The main point is that of six instances found, five were wrong. We want to optimize to catch the error.